ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixClusterManagerTest.java

```diff
@@ -699,6 +711,53 @@ public void xidTest() throws Exception {
     assertTrue(ignoreInstanceReplica.isDown());
   }
 
+  /**
+   * Ensure that effects of the listenCrossColo config is as expected. When it is set to false, the Helix cluster manager
+   * initializes fine, but listens to subsequent InstanceConfig changes in the local colo only.
+   */
+  @Test
+  public void listenCrossColoTest() {
+    assumeTrue(!useComposite);
+    HelixClusterManager helixClusterManager = (HelixClusterManager) clusterManager;
+    Counter instanceTriggerCounter = helixClusterManager.helixClusterManagerMetrics.instanceConfigChangeTriggerCount;
+    Map<String, HelixManager> helixManagerMap = helixClusterManager.getHelixManagerMap();
+    for (Map.Entry<String, HelixManager> entry : helixManagerMap.entrySet()) {
+      if (entry.getKey().equals(localDc)) {
+        assertTrue("Helix cluster manager should always be connected to the local Helix manager",
+            entry.getValue().isConnected());
+      } else {
+        assertEquals(
+            "Helix cluster manager should be connected to the remote Helix managers if and only if listenCrossColo is"
+                + "set to false", listenCrossColo, entry.getValue().isConnected());
```

Comment:
minor: should be `if and only if listenCrossColo is set to true`

Reply:
Addressed.